### DISCUSSION TITLE
Investigating: Not working on YouTube

### DISCUSSION THREAD

**Discussion by Canto007** [aclap-dev/video-downloadhelper#1041] [Help and Q&A]
### Your question

Hi DownloadHelper team
my licence isn't working anymore. It doesn't detect any videos.
I'm using it under Windows 10/Firefox, with and without VPN,same result.

Can you help please?
Below my license information and a screen capture
![thumbnail_image](https://github.com/user-attachments/assets/553cb6e9-d36e-44f8-a0ce-3022ddcedfb2)
![thumbnail_image 2](https://github.com/user-attachments/assets/ae0095b4-c7c4-4da4-8a2c-2c9cb4785dc4)

https://github.com/user-attachments/assets/25cf51d8-ddd0-4c94-b1c0-0ec6c2df071b

### Addon & CoApp version and details

![thumbnail_image](https://github.com/user-attachments/assets/a6800c30-55ff-4b71-beb8-68d41a6abfdc)
Version 9.0.2.12 (stable)
target Mozilla   locale: fr
Appli compagnon installée (v2.0.19)

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

I'm running Windows 11 Pro 64-bit version 23H2 build 22631.3880, Firefox 128.0 64-bit, licensed VDH 9.0.2.14a1 beta, CoApp 2.0.19.

When I visit . . .

https://www.youtube.com/watch?v=9Qgnp5RDlCM

I get this:

![#01](https://github.com/user-attachments/assets/f1236b90-8931-4f6c-aa99-b86bcb6b0363)

Perhaps the VDH beta would cure your problem.  I am certainly not seeing the same issue as you are.  If you want me to visit the same YouTube videos you have, you need to include their URLs in a post, as I have done in this post.

**Reply by Canto007** [reply]
I'm running Windows 10, and this beta version is not available.
I didn't provide YouTube link as VDH  is not working for me, for any video!
Still expecting support from VDH team, before claiming for refund ...

**Reply by GrampaWildWilly** [reply]
I don't know how you came to the conclusion that the beta version is not available for you.  Click the Wiki link at the top of any page of this forum.  That takes you to what amounts to the closest thing we have to a user manual, something you & everybody should get in the habit of referring to regularly.  Down the right side of that page is a table of contents.  Click the chapter about beta versions.  There is definitely a beta version of VDH for Firefox there.  It is available.  You can  find it.  Go on.  Do it.

**Reply by GrampaWildWilly** [reply]
It doesn't really matter whether you've been able to use VDH to download any given YouTube video.  Especially if you have not been able to download a particular YouTube video, you should give me its URL so I can test it.  As you can see in the example I provided, VDH works just fine for me on YouTube.  It's not out of the realm of possibility that you are looking at something that VDH can't handle.  Such things exist.  Let's see if you are hitting one, or a few.

**Reply by paulrouget** [reply]
> I didn't provide YouTube link as VDH is not working for me, for any video!

We need a URL. We can't "guess" what's not working for you.

**Comment by paulrouget** [comment]
@Canto007 

1. can you confirm it's not working **only** on youtube? For example, does it work here: https://hlsjs.video-dev.org/demo/ ?

2. what happens if you open this URL: 
https://www.youtube.com/watch?v=9Qgnp5RDlCM - does the icon stays grey?

**Reply by OneRolf** [reply]
For me, the same result as reported earlier. The videos 1 and 2 are recognized by VDH, and I can choose a variety to dl. DL popup pops up, but dl stops at 0%.
I'm wondering if something is the matter with my settings and setup, or of this is a general fault.

Logged in to YT
W7 Professional, Service Pack 1
Firefox 115.22.0esr (up to date 3 apr 2025)
VDH 9.3.5.2 beta (no new updates found 3 apr 2025)
CoApp 2.0.20.0
Sweden

**Comment by Canto007** [comment]
Yes Paul, it works with this link

Cordialement
Karim Zebiche
________________________________
De : Paul ***@***.***>
Envoyé : mercredi 31 juillet 2024 14:15
À : aclap-dev/video-downloadhelper ***@***.***>
Cc : Canto007 ***@***.***>; Mention ***@***.***>
Objet : Re: [aclap-dev/video-downloadhelper] Investigating: Not working on YouTube (Discussion #1041)


@Canto007<https://github.com/Canto007> can you confirm it's not working only on youtube? For example, does it work here: https://hlsjs.video-dev.org/demo/ ?

**Reply by paulrouget** [reply]
@Canto007 I gave you 2 links. I'd like to know which one works, which one didn't work.

**Comment by GrampaWildWilly** [comment]
With Windows 11 Pro 64-bit version 23H2 build 22631.3958, Firefox 128.0.3 64-bit, licensed VDH 9.1.0.39 beta, CoApp 2.0.19 I was able to download the bunny cartoon.  I've seen that one before.  I think somebody else reported it as being a problem quite a long time ago, like maybe a year.

I like your choice of YouTube test videos.  I was able to download that one as well.  However, it has happened in the past that people will be on here (or the old Google Groups forum) saying they're having trouble with YouTube but I'm here merrily bopping along no problem.  Then a couple of days later, whatever YouTube is doing finally lands here & I start seeing the same problems as everybody else.  My understanding is YouTube has a vast network of servers & when they make a change, it rolls across their network like a tsunami.  So it's quite possible that's happening again & I just haven't been swept away yet.  I must live in the most remote region of the YouTube network.  I haven't decided if I should feel lucky.

**Comment by Canto007** [comment]
Hi Paul

sorry for the ate response.
Surprisingly, as I didn't change anything, it's working today.
So it seems I can close my request 🙂

Imanswer your previous question, I'm using it on YouTube, under Firefox, under Windows 10  Version 22H2 for x64-based Systems (KB5040525)

Regards
Karim Zebiche
________________________________
De : Paul ***@***.***>
Envoyé : jeudi 1 août 2024 05:01
À : aclap-dev/video-downloadhelper ***@***.***>
Cc : Canto007 ***@***.***>; Mention ***@***.***>
Objet : Re: [aclap-dev/video-downloadhelper] Investigating: Not working on YouTube (Discussion #1041)


@Canto007<https://github.com/Canto007> I gave you 2 links. I'd like to know which one works, which one didn't work.

**Comment by SimonKeba** [comment]
I've got the same problem : in France, not any video YT detected... :/ i'm a registered user, mas0S 12.7.6, Firefox 136.0.4...
 https://hlsjs.video-dev.org/demo/ 
 works (VDH not grey)
https://www.youtube.com/watch?v=9Qgnp5RDlCM
 This one doesn't...(grey)
If you have any cue, i'd be delighted

**Reply by shawncaron** [reply]
Can confirm same issue on Arch Linux w/FF 137.0.  Other sites with videos work fine.  Just YT not working.

**Reply by brainode** [reply]
It happend few weeks ago, same for me
